app/pages/OrganizationListingPage.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState, useEffect } from 'react';
+import ReactMarkdown from 'react-markdown';
+import { useParams } from 'react-router-dom';
+import { Helmet } from 'react-helmet-async';
+import {
+  AddressInfo,
+  PhoneNumber,
+  ResourceCategories,
+  Website,
+  Email,
+  ActionSidebar,
+  MobileActionBar,
+  MOHCDBadge,
+  RelativeOpeningTime,
+  Notes,
+  Services,
+  TableOfOpeningTimes,
+} from 'components/listing';
+import { MapOfLocations } from 'components/maps';
+import { Loader } from 'components/ui';
+import { getSiteTitle } from '../utils/whitelabel';
+import { fetchOrganization, getResourceLocations, Organization } from '../models';
+
+// Page at /organization/123
+export const OrganizationListingPage = () => {
+  const { id } = useParams<{ id: string }>();
+  const [org, setOrg] = useState<Organization|null>(null);
+
+  useEffect(() => {
+    fetchOrganization(id as any)
```

Comment:
You're right, I resisted doing this because it just felt like boilerplate to parseInt on every param. I think the best approach is to just accept ID as a string in the fetch operations, since it is only ever called from URL params as strings.

Reply:
Yeah, I agree with leaving it as a string, and just making sure that the types are updated to reflect its real type.